lib/chewy/index.rb

```diff
@@ -152,6 +152,14 @@ def self.scopes
       public_methods - Chewy::Index.public_methods - type_names.map(&:to_sym)
     end
 
+    def self.timestamp_ordered_import(flag = nil)
+      if flag.nil?
+        @timestamp_ordered_import == true
+      else
+        @timestamp_ordered_import = flag
+      end
+    end
+
```

Comment:
There is one problem with this: all inheritors of `Chewy::Index` will share the same `@timestamp_ordered_import` variable.